### DISCUSSION TITLE
Presets confuse me

### DISCUSSION THREAD

**Discussion by florenso** [microsoft/vscode-cmake-tools#3993] [Q&A] [answered]
I want to setup my project so that configuration has two options: The platform(windows/linux/arm64) and the type(Debug/Release).
after that my project has multiple targets i can build, a.lib ttest.exe, b.lib, btest.exe, etc...

But if i create a CMakePresets.json the second options for config goes away. and i get an extra option under Build.
cant i use kits and presets together?

![image](https://github.com/user-attachments/assets/3f4eb292-d233-49b5-aab9-ed72280f32cb)

**Comment by gcampbell-msft** [comment, marked as answer]
@florenso No, Kits and Presets are separate design ideas for how to control project setup and configuration. CMake Presets is the suggested path forward as it has official support from both our team and Kitware (developers of cmake.exe) as well
